account/identity.go

```diff
@@ -21,6 +22,11 @@ import (
 const (
 	// KeycloakIDP is the name of the main Keycloak Identity Provider
 	KeycloakIDP string = "kc"
+
+	IDENTITY_RESOURCE_TYPE_ORGANIZATION = "identity/organization"
+	IDENTITY_RESOURCE_TYPE_TEAM         = "identity/team"
+	IDENTITY_RESOURCE_TYPE_GROUP        = "identity/group"
+	IDENTITY_RESOURCE_TYPE_USER         = "identity/user"
```

Comment:
Let's use camelcase as a Go convention?
https://stackoverflow.com/a/22688926/823386 unless we have a reason not to :)

Reply:
Interesting, I didn't realise that Go broke established norms for constant naming conventions. ;) I've updated to use camel case.